Tests/Pcap++Test/Common/PcapFileNamesDef.h

```diff
@@ -24,3 +24,4 @@
 #define EXAMPLE_LINKTYPE_IPV6 "PcapExamples/linktype_ipv6.pcap"
 #define EXAMPLE_LINKTYPE_IPV4 "PcapExamples/linktype_ipv4.pcap"
 #define EXAMPLE_SOLARIS_SNOOP "PcapExamples/solaris.snoop"
+#define SLL2_PCAP_PATH "PacketExamples/Sll2Packet.pcap"
```

Comment:
1. There is no newline at the end of this file, hence the pre-commit tests fail: https://github.com/seladb/PcapPlusPlus/actions/runs/5705464095/job/15471210474?pr=1138
2. `PacketExamples/Sll2Packet.pcap` is under `Packet++Test`, not under `Pcap++Test`, that is why all the tests fail: https://github.com/seladb/PcapPlusPlus/actions/runs/5705464095?pr=1138 . What I'd suggest is:
    - Move `Sll2Packet.pcap` to `Pcap++Test/PcapExamples/` and use it in this test
    - Then take the single packet we use in `Packet++Test` (`Sll2Packet.dat`), convert it to pcap and put this pcap in `Packet++Test/PacketExamples`

Reply:
Fixed in these commits:
- https://github.com/seladb/PcapPlusPlus/pull/1138/commits/02ddd9ad7f9c90d94465c4e73a3fe518fd63b0cc
- https://github.com/seladb/PcapPlusPlus/pull/1138/commits/d9c10a18b47f17805c6320bf7a2fefe271dafef4